ZeldaFullEditor/ZSCustomOverworld.asm

```diff
@@ -3284,18 +3934,21 @@ InitColorLoad2:
     .storeColor
 
     ; Set transparent color.
-    STA.l $7EC300 : STA.l $7EC340
+    STA.l $7EC300
+    STA.l $7EC340
 
-    ; TODO: Based on the conditions as explained above, double check that this is not
-    ; needed for any of them.
-    ;STA.l $7EC500 : STA.l $7EC540
+    ; TODO: Based on the conditions as explained above, double check that this is
```

Comment:
The buffers (300, 340) get copied into cgram (500, 540) inside of `Overworld_CopyPalettesToCache` so likely not needed unless someone is doing custom color adjustments and needs to operate on cgram.